test/ansible/playbooks/reconciliation.yml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.kubernetes
+  tasks:
+    - name: retrieve configmap
+      k8s_info:
+        api_version: v1
+        kind: ConfigMap
+        namespace: '{{ meta.namespace }}'
+        name: '{{ meta.name }}'
+      register: configmap
+
+    - name: create configmap
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: '{{ meta.name }}'
+            namespace: '{{ meta.namespace }}'
+          data:
+            iterations: '1'
+      when: configmap.resources|length == 0
+
+    - name: Update ConfigMap
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: '{{ meta.name }}'
+            namespace: '{{ meta.namespace }}'
+          data:
+            iterations: '{{ (configmap.resources.0.data.iterations|int) + 1 }}'
+      when: configmap.resources|length > 0 and (configmap.resources.0.data.iterations|int) < 6
+
+    - name: Using the requeue_after module
+      operator_sdk.util.requeue_after:
+        time: 1s
```

Comment:
You may want to set the same `when: ` condition as above on this task, so it stops requeueing when the test is done